evaluate_tflite.py

```diff
@@ -2,156 +2,11 @@
-            daemon=True,
-            name="tflite_process_{}".format(i),
-        )
-        worker_process.start()  # Launch reader() as a separate python process
-        processes.append(worker_process)
-
-    print([x.name for x in processes])
-
-    wavlist = []
-    ground_truths = []
-    predictions = []
-    losses = []
-    wav_filenames = []
-
-    with open(args.csv, "r") as csvfile:
-        csvreader = csv.DictReader(csvfile)
-        count = 0
-        for row in csvreader:
-            count += 1
-            # Relative paths are relative to the folder the CSV file is in
-            if not os.path.isabs(row["wav_filename"]):
-                row["wav_filename"] = os.path.join(
-                    os.path.dirname(args.csv), row["wav_filename"]
-                )
-            work_todo.put(
-                {"filename": row["wav_filename"], "transcript": row["transcript"]}
-            )
-            wav_filenames.extend(row["wav_filename"])
-
-    print("Totally %d wav entries found in csv\n" % count)
-    work_todo.join()
-    print("\nTotally %d wav file transcripted" % work_done.qsize())
-
-    while not work_done.empty():
-        msg = work_done.get()
-        losses.append(0.0)
-        ground_truths.append(msg["ground_truth"])
-        predictions.append(msg["prediction"])
-        wavlist.append(msg["wav"])
-
-    # Print test summary
-    _ = calculate_and_print_report(
-        wav_filenames, ground_truths, predictions, losses, args.csv
-    )
-
-    if args.dump:
-        with open(args.dump + ".txt", "w") as ftxt, open(
-            args.dump + ".out", "w"
-        ) as fout:
-            for wav, txt, out in zip(wavlist, ground_truths, predictions):
-                ftxt.write("%s %s\n" % (wav, txt))
-                fout.write("%s %s\n" % (wav, out))
-            print("Reference texts dumped to %s.txt" % args.dump)
-            print("Transcription   dumped to %s.out" % args.dump)
-
-
-def parse_args():
-    parser = argparse.ArgumentParser(description="Computing TFLite accuracy")
-    parser.add_argument(
-        "--model", required=True, help="Path to the model (protocol buffer binary file)"
-    )
-    parser.add_argument(
-        "--scorer", required=True, help="Path to the external scorer file"
-    )
-    parser.add_argument("--csv", required=True, help="Path to the CSV source file")
-    parser.add_argument(
-        "--proc",
-        required=False,
-        default=cpu_count(),
-        type=int,
-        help="Number of processes to spawn, defaulting to number of CPUs",
-    )
-    parser.add_argument(
-        "--dump",
-        required=False,
-        help='Path to dump the results as text file, with one line for each wav: "wav transcription".',
-    )
-    args, unknown = parser.parse_known_args()
-    # Reconstruct argv for absl.flags
-    sys.argv = [sys.argv[0]] + unknown
-    return args
-
-
 if __name__ == "__main__":
-    main(parse_args())
+    try:
```

Comment:
This script should also inform users that they should use the module syntax with `python -m` instead of calling the script directly. See `train.py` on the root dir for an example of what I mean.